### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jogo-**

### CODE REVIEW

File: src/mame/machine/amstrad.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ The Z80 must also access the same ram.
 To maintain the screen display, the Z80 is halted on each memory access.
 
 The result is that timing for opcodes, appears to fall into a nice pattern,
-where the time for each opcode can be measured in NOP cycles. NOP cycles is
+where the time for each opcode can be measured in NOP cycles. NOP cycle is
```
Comment (cuavas) [inline src/mame/machine/amstrad.cpp:29]: Plural is correct here – a NOP takes multiple cycles to complete.

File: src/mame/machine/amstrad.cpp [hunk 2]
```diff
@@ -3230,7 +3230,7 @@ MACHINE_RESET_MEMBER(amstrad_state,kccomp)
 	amstrad_common_init();
 	kccomp_reset_machine();
 
-	/* bit 1 = /TEST. When 0, KC compact will enter data transfer
+	/* bit 1 = /TEST. When 0, KC Compact will enter data transfer
```
Comment (cuavas) [inline src/mame/machine/amstrad.cpp:3233]: This is inconsistent with the other place you changed to “KC compact”.  If you look at pictures of the machine, the label actually does say “KC compact” with a lowercase “c”.

File: src/mame/machine/amstrad.cpp (hunk 1 above)
Comment (jogo-) [reply src/mame/machine/amstrad.cpp:29]: Agreed

File: src/mame/machine/amstrad.cpp (hunk 2 above)
Comment (jogo-) [reply src/mame/machine/amstrad.cpp:3233]: Correct. I can't remember why I changed only this one.
Do you need me to fix both of your remarks with a new PR ?

Comment (rb6502) [reply src/mame/machine/amstrad.cpp:3233]: Yes, please make a new PR, and give it a more descriptive title.

**Pull request merged**